### PR TITLE
New Icons August

### PR DESCRIPTION
**Pull request by conradennis**
- Added 22 net new icons
- Updated 1 icon
- Renamed/categorized 1 icon
- Updated ui-icon-master.ai
- Updated categories.yml

#### Changelog

**New**

- denominate.svg
- earth.svg
- earth--americas.svg
- earth--americas--filled.svg
- earth--europe-africa.svg
- earth--europe-africa--filled.svg
- earth--filled.svg
- earth--southeast-asia.svg
- earth--southeast-asia--filled.svg
- folder--off.svg
- logo--glassdoor.svg
- nominate.svg
- rotate--clockwise.svg
- rotate--clockwise--alt.svg
- rotate--clockwise--alt--filled.svg
- rotate--clockwise--filled.svg
- rotate--counterclockwise.svg
- rotate--counterclockwise--alt.svg
- rotate--counterclockwise--alt--filled.svg
- rotate--counterclockwise--filled.svg
- SDK.svg
- sunrise.svg

**Changed**

- fog.svg --> cloud--app.svg
- sunset.svg
- ui-icon-master.ai
- categories.yml

### CODE REVIEW

File: packages/icons/metadata.yml [hunk 1]
```diff
@@ -833,6 +833,14 @@ icons:
         categories:
           - name: Organization
             subcategory: File
+      - name: folder--off
```
Comment (joshblack) [inline packages/icons/metadata.yml:836]: @chrisconnors-ibm do we want to add metadata in this pass or hold off till later? Meaning stuff like aliases/friendly names/etc

**Review by joshblack — APPROVED** [review]

**Comment by joshblack** [comment]
Just a note to anyone looking to merge, make sure the commit message is a `feat(icons):` 👍

File: packages/icons/metadata.yml (hunk 1 above)
Comment (chrisconnors-ibm) [reply packages/icons/metadata.yml:836]: they have to have a category/subcategory; have to have a friendly name. In theory synonyms could wait, but Strongly Recommend they be included.

Comment (chrisconnors-ibm) [reply packages/icons/metadata.yml:836]: they have to have a category/subcategory; have to have a friendly name. In theory synonyms could wait, but Strongly Recommend they be included.

Comment (joshblack) [reply packages/icons/metadata.yml:836]: @chrisconnors-ibm do we want this included in the next release? If so, who should I track down to help with contributing the appropriate meta info?

**Review by chrisconnors-ibm — APPROVED** [review]
lgtm - need to populate synonyms as a follow on issue

**Comment by chrisconnors-ibm** [comment]
(we should figure out how to get these to trigger creating a new IDL or Icons Libraries - manually at first)

**Pull request merged**